src/scss/partials/header.scss

```diff
@@ -1,214 +1,251 @@
 .header {
-	display: flex;
-	position: relative;
-	align-content: center;
-	align-items: center;
-	justify-content: space-between;
-	padding: 20px;
-	border-bottom: 10px solid var.$blue;
-	background: var.$white;
-	color: var.$red;
-
-	.logo {
-		display: block;
-		width: 200px;
-		margin-right: auto;
-
-		img, svg {
-			max-width: 100%;
-		}
-	}
-
-	nav {
-		position: relative;
-		z-index: 50;
-		font-size: 1.4em;
-		font-weight: 600;
-		text-align: center;
-
-		&.open {
-			ul {
-				display: block;
-			}
-		}
-
-		ul {
-			display: none;
-			position: absolute;
-			z-index: 150;
-			top: 100%;
-			right: 0;
-			width: 90vw;
-			max-width: 400px;
-			max-height: calc(100vh - 150px);
-			overflow-y: auto;
-			border-radius: 12px;
-			background: var.$white;
-			box-shadow: 0 1px 3px rgba(var.$neutral, 0.15), 0 0 8px rgba(var.$neutral, 0.1);
-			color: var.$blue;
-		}
-
-		a {
-			padding: 0.6em;
-
-			&:hover {
-				background: var.$blue;
-				color: var.$white;
-				text-decoration: none;
-			}
-
-			.label {
-				position: static;
-				margin-left: 1ch;
-			}
-		}
-
-		li {
-			&.current_page_item {
-				> a {
-					background: var.$red;
-					color: var.$white;
-				}
-			}
-
-			&.cta {
-				background: var.$blue;
-				color: var.$white;
-			}
-		}
-	}
+  display: flex;
+  position: relative;
+  align-content: flex-start;
```

Comment:
Try going through and removing reach of these, and check if the styles still look right, they may not be needed